ci/Jenkinsfile-test

```diff
@@ -92,11 +129,21 @@ pipeline {
               dir("std") {
                 deleteDir()
               }
-              checkout changelog: false, poll: false, scm: [$class: 'GitSCM', branches: [[name: '*/master']], doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'RelativeTargetDirectory', relativeTargetDir: 'std']], submoduleCfg: [], userRemoteConfigs: [[credentialsId: 'inmantaci', url: 'https://github.com/inmanta/std.git']]]
```

Comment:
This change is to make sure we use the correct branch for the v1 std module:
- for core branch `iso8` and `iso9`  -> use std branch `master`
- for core branch `iso7` -> use std branch `iso7`